Feature/meta styling

narrowdesign opened the pull request.

forresto commented on src/components/fold-media.css:
```diff
@@ -0,0 +1,7 @@
+.FoldMedia .AttributionEditor-metadata {
+  top: -75px;
+}
```
😾 Why hide the image?

narrowdesign replied:
I started with what @d4tocchini has here and scaled way back on the overlap: 
![image](https://cloud.githubusercontent.com/assets/440221/14207899/bd373e4e-f7ea-11e5-999d-4a908f8da15c.png)

forresto replied:
If the app is animating the cover image, we'll need an option when mounting to hide the fold cover, and let the app take care of the overlap. That will complicate the new post flow though (for apps).

forresto replied:
That's only the web app. Mobile integrations / smaller screens probably don't want the overlap. So, kill it?

narrowdesign replied:
True. I'm not attached to the overlap. Do want to stay consistent with D4's intention on this and the animation. What do you say @d4tocchini?

forresto commented:
Can we put the links in the same grey box as title, description?

![screen shot 2016-03-31 at 5 51 58 pm](https://cloud.githubusercontent.com/assets/395307/14192048/5ee51a72-f769-11e5-8e7d-0347e33aaae3.png)

forresto commented:
@narrowdesign ideas here?

![links overlap](https://cloud.githubusercontent.com/assets/395307/14212789/8d2a8748-f802-11e5-8d1e-28b7075e9685.png)

narrowdesign commented:
fixed @forrest. I just changed links to relative position and removed the bottom padding. Fixed the double line too.

![image](https://cloud.githubusercontent.com/assets/440221/14216325/30100880-f816-11e5-93c5-b0ba657909da.png)

forresto commented:
@narrowdesign what do you think about these tweaks? Neg margins on links, no border, uppercase.

![screen shot 2016-04-01 at 3 49 49 pm](https://cloud.githubusercontent.com/assets/395307/14218348/9537ef6a-f821-11e5-9088-af537abc5ca9.png)

narrowdesign commented:
@forresto I'm cool with that visually (except the twitter border color is darker) but those links are confusing me. I still don't know what they mean unless I really try. Maybe they need labels. Let's run with this and I'll play about it.
